apps/backend/src/websocket/handlers/field.ts

```diff
@@ -220,8 +220,12 @@ export const handleUpdateMatchTeams = async (
   });
 
   callback({ ok: true });
+
+  const oldMatch = match;
   match = await db.getMatch({ _id: new ObjectId(matchId) });
   namespace.to('field').emit('matchUpdated', match);
+  if (match.scheduledTime != oldMatch.scheduledTime)
```

Comment:
At least one participant team has been updated. Emit an event for every participant team that was updated. No if statement needed